Baseball/Team Building.user.js

```diff
@@ -54,12 +55,12 @@ function RandomText(length) {
 	return text;
 }
 
-async function IncreasePlayer(id, type, playerID, verify) {
-	[...document.querySelectorAll('.Up')].forEach(buttonTag => {
+async function IncreasePlayer(id, type, playerID, verify, buttonId) {
+	[...document.querySelectorAll('.Up0'), ...document.querySelectorAll('.Up1')].forEach(buttonTag => {
```

Comment:
`querySelector` and `querySelectorAll` selects like CSS so you can just do `'.Up0, .Up1'` instead of splitting it up into two.

Reply:
Fixed